src/app/components/NavBar.tsx

```diff
@@ -343,7 +344,7 @@ function NavbarItemMobile({
           active ? "text-[#cafc40]" : "text-white"
         }`}
       >
-        {title}
+        {translate(title)}
```

Comment:
Please change to follow our conventions:
```suggestion
        {t(title)}
```